server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java

```diff
@@ -73,6 +76,25 @@ public LoggerContextListener enableJulChangePropagation(LoggerContext loggerCont
     return propagator;
   }
 
+  public void configureRootLogger(LoggerContext ctx, Props props, String threadIdFieldPattern, String fileName) {
+    String logFormat = LOG_FORMAT.replace(THREAD_ID_PLACEHOLDER, threadIdFieldPattern);
+    // configure appender
+    LogbackHelper.RollingPolicy rollingPolicy = createRollingPolicy(ctx, props, fileName);
+    FileAppender<ILoggingEvent> fileAppender = rollingPolicy.createAppender("file");
+    fileAppender.setContext(ctx);
+    PatternLayoutEncoder fileEncoder = new PatternLayoutEncoder();
+    fileEncoder.setContext(ctx);
+    fileEncoder.setPattern(logFormat);
+    fileEncoder.start();
+    fileAppender.setEncoder(fileEncoder);
+    fileAppender.start();
+
+    // configure logger
+    Logger rootLogger = ctx.getLogger(ch.qos.logback.classic.Logger.ROOT_LOGGER_NAME);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "rootLogger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this fully qualified name with "Logger" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1942)